src/middlewares/authMiddleware.js

```diff
@@ -0,0 +1,23 @@
+import { jwtService } from '../services/jwt.service.js';
+
+export const authMiddleware = (req, res, next) => {
+  const authorization = req.headers['authorization'] || '';
+  const [, token] = authorization.split(' ');
+
+  if (!authorization || !token) {
+    res.status(401).send('Token is not valid');
```

Comment:
The error message 'Token is not valid' is used both when the token is missing and when it is invalid. Consider using a more specific message for missing tokens, such as 'Authorization token is missing'.